tests/main/interfaces-udisks2/task.yaml

```diff
@@ -4,8 +4,8 @@ details: |
     The udisks2 interface allows operating as or interacting with the UDisks2 service
 
 # Interfaces not defined for ubuntu core systems
-# FIXME: fails in debian-sid for unknown reasons
-systems: [-ubuntu-core-*, -debian-sid-*]
+# FIXME: fails in debian-sid and ubuntu-20.04 for unknown reasons
+systems: [-ubuntu-core-*, -debian-sid-*, -ubuntu-20.04-*]
```

Comment:
Can we have a short retry loop around 'test-snapd-udisks2.udisksctl mount ..' instead? Apparently there is a race with the device becoming registered by udisks2? Alternatively, a retry loop around "test-snapd-udisks2.udisksctl  dump | MATCH $device" before mounting it, to wait for the device to be visible for udisks.

Reply:
Sure, thanks for the suggestion

Reply:
@stolowski the test has been fixed, thanks for the suggestion